cocos/spine/skeleton.ts

```diff
@@ -815,9 +882,16 @@ export class Skeleton extends UIRenderer {
     /**
      * @engineInternal
      */
-    public requestDrawData (material: Material, indexOffset: number, indexCount: number) {
+    public requestDrawData (material: Material, texureID: number, indexOffset: number, indexCount: number) {
         const draw = this._drawList.add();
         draw.material = material;
+        if (texureID === 0) {
+            draw.texture = this._textures[0];
```

Comment:
this._textures is an array now, when will the `index > 0` be used? Do we only use the first element now?

Reply:
yes

Reply:
Is it possible to use more than 1 texture in the future?